Makefile

```diff
@@ -44,6 +46,18 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
 	$(TOOLBIN)/helm install fybrik charts/fybrik --values $(VALUES_FILE) $(HELM_SETTINGS) \
                --namespace $(KUBE_NAMESPACE) --wait --timeout 120s
 
+.PHONY: deploy_old_CRD_version
+deploy_old_CRD_version: export VALUES_FILE?=charts/fybrik/values.yaml
+deploy_old_CRD_version: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
+	$(TOOLBIN)/kubectl create namespace $(KUBE_NAMESPACE) || true
+
+	$(TOOLBIN)/helm repo add fybrik-charts $(FYBRIK_CHARTS)
+	$(TOOLBIN)/helm repo update
+	$(TOOLBIN)/helm install fybrik-crd charts/fybrik-crd  \
+               --namespace $(KUBE_NAMESPACE) --version $(OLD_CRD_VERSION) --wait --timeout 120s
```

Comment:
I don't understand this command. If charts/fybrik-crd is used then we are installing the latest version, not the old one. For the old version the packaged charts should be used, shouldn't they?

Reply:
Correct, the old release version should be used.